### ISSUE TITLE
Improve NAT configuration

### ISSUE THREAD

**Issue by TiagoJacobs** [bigbluebutton/bbb-install#574]
Sometimes the NAT configuration / external IP detection fails - it happened to me this morning.

The only way of getting the setup done was manually overriding the `external_ip` variable within the `bbb-install.sh` script (maybe a DNS resolution problem).

It would be good if we can specify the external nat ip address as a parameter in the bbb-install call (with a new parameter `-n`).

Something like this (`a.b.c.d` is the external nat ip):
```
wget -qO- https://ubuntu.bigbluebutton.org/bbb-install-2.5.sh | bash -s -- -v focal-250 -s bbb.example.com -e info@example.com -w -n a.b.c.d
```

This would allow more flexibility, for instance, to use an external API:
```
wget -qO- https://ubuntu.bigbluebutton.org/bbb-install-2.5.sh | bash -s -- -v focal-250 -s bbb.example.com -e info@example.com -w -n `curl -s https://api.ipify.org`
```

PS: I'll send a PR for this soon, just writing the idea here for discussion.

**Comment by hahaxd35** [comment]
yes never worked perfect, big problem when server know public ip 
i recommend using "sed" commands that can be run atfer every update/upgrade
